12-offer-encoding.md

```diff
@@ -348,6 +473,8 @@ painful to have to special-case the "only one left" offer generation.
 
 Offers can be used to simply send money without expecting anything in return (tips, kudos, donations, etc), which means the description field is optional (the `offer_issuer` field is very useful for this case!); if you are charging for something specific, the description is vital for the user to know what it was they paid for.
 
+We insist that recurring requests be in order (thus, if you pay an invoice for #34 of a recurring offer, it implicitly commits to the successful payment of #0 through #33).  The `invreq_recurrence_paywindow` constrains how far you can pay in advance precisely, and if it isn't in the offer the defaults provide some slack, without allowing commitments into the far future.
```

Comment:
I think `invreq_recurrence_paywindow` is a typo to `offer_recurrence_paywindow`.

if not is missing in the TLVs and I don't get how it would work 👀 . `invreq_recurrence_paywindow` would be set by the payer but it would be controlled by the issuer

Reply:
Good catch.

Reply:
@rustyrussell I think this one might slipped out from your changes

Reply:
"This time for sure!"